cli/command/container/restart.go

```diff
@@ -48,6 +56,17 @@ func runRestart(dockerCli command.Cli, opts *restartOptions) error {
 		timeout = &timeoutValue
 	}
 
+	if opts.nAll {
```

Comment:
I think it will be more readable like that:
```Go
if opts.nAll {
    var err error
    opts.containers, err = dockerCli.Client().ContainerList(context.Background(), types.ContainerListOptions{})
    if err != nil {
        errs = append(errs, err.Error())
    }
}
```

In general, if you are passing `--all` you would like to set `opts.containers` to all containers that are running. Thus, there is no need to save it in a different variable and then extend `opts.containers`.